internal/platform/stat_test.go

```diff
@@ -153,36 +253,56 @@ func Test_StatFile_times(t *testing.T) {
 
 func TestStatFile_dev_inode(t *testing.T) {
 	tmpDir := t.TempDir()
+	d, err := os.Open(tmpDir)
+	require.NoError(t, err)
+	defer d.Close()
 
-	path1 := path.Join(tmpDir, "1")
+	path1 := pathutil.Join(tmpDir, "1")
 	f1, err := os.Create(path1)
 	require.NoError(t, err)
+	defer f1.Close()
 
-	path2 := path.Join(tmpDir, "2")
+	path2 := pathutil.Join(tmpDir, "2")
 	f2, err := os.Create(path2)
 	require.NoError(t, err)
+	defer f2.Close()
+
+	pathLink2 := pathutil.Join(tmpDir, "link2")
+	err = os.Symlink(path2, pathLink2)
+	require.NoError(t, err)
+	l2, err := os.Open(pathLink2)
+	require.NoError(t, err)
+	defer l2.Close()
 
+	// First, stat the directory
 	var stat1 Stat_t
+	require.NoError(t, StatFile(d, &stat1))
+	requireDirectoryDevIno(t, stat1)
```

Comment:
missing test. before we had no idea directories weren't returning inodes on windows